elisp/core.el

```diff
@@ -217,4 +217,6 @@
                             mal-nil))
                          (t
                           mal-nil))))))
+
+    (elisp-eval . ,(mal-fn (lambda (string) (mal-string (format "%S" (eval (read (mal-value string))))))))
     ))
```

Comment:
I'd like to have this as well, but it's problematic:

- `nil` and `()` are equivalent, therefore it's impossible to know which is meant on the MAL side
- loads of not representable types, such as buffers, bit arrays, markers, etc.

I recall complaining about it on `#mal` and getting the suggestion of returning a string with the printed representation instead, so that's why I'm doing this here.